### ISSUE TITLE
Documentserver running normally，but accessing Domo's /editor does not open the file.

### ISSUE THREAD

**Issue by Mercer08572** [ONLYOFFICE/docs-integration-sdk-java#36]
I started the DocumentServer using Docker.
The version of the DocumentServer is 8.2.2.
I can normally open and edit Word files through the welcome page of the DocumentServer, but I cannot access Word files through the JavaDemo's /editor.
Is it a configuration issue or some other reason?
the error image:
![image](https://github.com/user-attachments/assets/595798a7-7dba-42aa-a58f-1a3cf200afd3)

**Comment by SergeyKorneyev** [comment]
Hello,
Please check the Document Server logs to see if there are any errors that could help identify the cause of the issue. The warning message in the editor usually means that the callback handler could not be reached: https://api.onlyoffice.com/docs/docs-api/more-information/troubleshooting/#could-not-be-saved

**Comment by Mercer08572** [comment]
Thank you very much for your advice. I found the cause of the problem through the Document Server log. In Docker, the Document Server is accessing the demo interfaces based on the IP I input into the browser's address bar. I used localhost and the loopback address on my machine before, which caused the Document Server in Docker to be unable to access the host machine. Using the local area network address of the host machine solved the problem.
My English is not good, I'm not sure if I have expressed myself clearly. In the end, I still want to thank you.
